packages/config/config/devices/0x0129/ykfcon.json

```diff
@@ -29,7 +34,7 @@
 			"$import": "templates/yale_template.json#auto_relock"
 		},
 		"3": {
-			"$import": "templates/yale_template.json#auto_relock_time_180"
+			"$import": "templates/yale_template.json#manual_relock_time"
```

Comment:
Did this change?
/cc @blhoward2

Reply:
As per documentation, param 3 is 7-60 with default value of 7. This matches the “manual_relock_time” template.
http://manuals-backend.z-wave.info/make.php?lang=en&sku=YKFCON&cert=ZC10-18015918

Reply:
This looks right but that parameter is inconsistent in the template itself. It should be renamed to Auto Relock Time to match the others and the description should be removed.

Reply:
This manual also mentions the auto relock time of max. 180:
https://products.z-wavealliance.org/ProductManual/File%3Ffolder%3D%26filename%3DMarketCertificationFiles/3211/YaleRealLivingAssureSL_Z-WavePlus_SystemIntegratorsGuide.pdf

Could this be the difference between Yale's Z-Wave Module 1 and 2?